app/src/main/java/net/opendasharchive/openarchive/services/webdav/WebDAVSiteController.kt

```diff
@@ -127,6 +131,24 @@ class WebDAVSiteController : SiteController {
 
         if (sardine == null) throw IOException("client not init'd")
 
+        val protocols: List<Protocol?> = object : ArrayList<Protocol?>() {
+            init {
+                add(Protocol.HTTP_1_1)
+            }
+        }
+
+        val okHttpClient: OkHttpClient = OkHttpClient.Builder()
```

Comment:
Why the boilerplate? Do you not trust the type inference?

Reply:
Updated!